specs/scoring_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'spec_helper'
+require_relative '../lib/scoring'
+
+describe Scrabble::Scoring do
+  describe "#score" do
+    it "Calculates the score of one word" do
+      Scrabble::Scoring.score("apple").must_equal 9
+    end
+
+    it "Calculates the score of word without case sensitivity" do
+      Scrabble::Scoring.score("APPLE").must_equal 9
+    end
+
+    it "Raises error if input is not a string" do
+      proc {
+        Scrabble::Scoring.score(1)
+      }.must_raise ArgumentError
+    end
+
+    it "Gives 50 bonus points for 7 letter words" do
+      Scrabble::Scoring.score("chicken").must_equal 68
+    end
+  end
+
+  describe "#highest_score_from" do
+    it "Returns the word with the highest score" do
+      words = ["chicken", "apple"]
+      Scrabble::Scoring.highest_score_from(words).must_equal "chicken"
+    end
+
+    it "Returns word with fewer letters in a tie" do
```

Comment:
You should also probably be testing to make sure order doesn't affect the word selected if the words are of different length.